pages/price-feeds/index.mdx

```diff
@@ -14,11 +14,7 @@ Pyth offers two different versions of price feeds:
   several [Cosmos chains](price-feeds/use-real-time-data/cosmwasm.md), [Aptos](price-feeds/use-real-time-data/aptos.md), [Sui](price-feeds/use-real-time-data/sui.md), and [NEAR](price-feeds/use-real-time-data/near.md).
   More ecosystems are coming soon!
 
+To learn more about Pyth price feeds and to start using them, follow the [Quick Start](quick-start.md) guide.
```

Comment:
isn't this supposed to say Getting Started and point to getting-started.md?

(minor comment) I suggest making the sentence an imperative: "Follow the [Getting Started] guide to learn more about Pyth and integrate Pyth Price Feeds into your application". This slight rewrite will help with clarity, because it puts the action you want them to take front-and-center.

Reply:
Oh, I see what you mean. 👍